zig-aarch64.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+zig cc -target aarch64-unknown-linux-gnu $@
```

Comment:
```suggestion
# shellcheck disable=SC2068
zig cc -target aarch64-unknown-linux-gnu $@
```

to disable the [`shellcheck`](https://github.com/cargo-bins/cargo-quickinstall/actions/runs/3915594296/jobs/6693920222#step:3:12) here.